snow/consensus/snowman/consensus_test.go

```diff
@@ -937,110 +919,6 @@ func RecordPollTransitiveVotingTest(t *testing.T, factory Factory) {
 	require.Equal(choices.Rejected, block4.Status())
 }
 
-func RecordPollDivergedVotingTest(t *testing.T, factory Factory) {
```

Comment:
Mentioned in https://github.com/ava-labs/avalanchego/pull/2949/commits/3e9e5608d4d22faed649afbe8e0a59ba150969b4 - this test requires divergent betas so that in the binary decomposition of voting, it's possible to conditionally accept an intermediate node and then transitively apply a vote through a block that has already been pruned due to the accepted prefix.